backend/development.ini.sample

```diff
@@ -870,7 +870,12 @@ caldav.radicale.headers.Access-Control-Expose-Headers = Etag
 ## Jitsi Meet configuration parameters
 ; call.jitsi_meet.url =
 # example:
-; call.jitsi_meet.url = https://meet.jit.si
+# call.jitsi_meet.url = https://meet.jit.si
+
+## Time (in ms) before the call is considered unanswered
+; call.unanswered_timeout =
```

Comment:
Tracim not have default value ?

If default value exist, good syntax is `; call.unanswered_timeout = 30000`

Reply:
Indeed, Tracim got a default value of 30 seconds. I'll change the syntax.

Reply:
Indeed, Tracim got a default value of 30 seconds. I'll change the syntax.